stix_shifter_modules/aws_athena/stix_transmission/query_connector.py

```diff
@@ -35,10 +37,14 @@ def create_query_connection(self, query):
                 query = json.loads(query)
             query_service_type = list(query.keys())[0]
             config_details = service_types[query_service_type]
+            if all([True if config not in self.connection.keys() else False for config in config_details]):
+                random_id = ''.join([random.choice(string.ascii_letters + string.digits) for _ in range(30)])
+                return_obj = {'success': True, 'search_id': 'athena-' + random_id.lower() + ':' + query_service_type}
```

Comment:
I would make the id more sense like add dummy in the search id instead of Athena such as `a9afe0ac-b6cf-4569-a4eb-730bd6cdfd5e-dummy:guardduty`

Reply:
@mdazam1942 The "Athena" prefix is now removed and the dummy search ID is suffixed with the string "dummy". The changes are updated to PR branch.